brave/brave-core
# Design system button

Introduces:
- Design System button based on the one in Figma. Includes "primary" and "secondary" levels, as well as all sizes from "tiny" to 'jumbo". Does _not_ include "colored" theme version or "tertiary" variants as well as the separate icon-only buttons.
- CSS modules

Fixes button in NTP customize panel, specifically for Brave News RSS feed adding.

Resolves https://github.com/brave/brave-browser/issues/20639

<img width="442" alt="image" src="https://user-images.githubusercontent.com/741836/155626087-56622a49-be6d-4fac-a1ab-8e4e565b9410.png">
<img width="485" alt="image" src="https://user-images.githubusercontent.com/741836/155626111-0d4d4ea9-9081-40e4-acb7-00722eab50b7.png">


## Test Plan:
In storybook or in real browser with Brave News enabled:
- Open NTP
- Open Customize panel
- Switch to Brave News tab
- Enter an rss feed url
- Observe button states for light and dark mode:
    - hover
    - focus
    - click
    - loading

## Submitter Checklist:

- [x] I confirm that no security/privacy review [is needed](https://github.com/brave/brave-browser/wiki/Security-reviews), or that I have [requested](https://github.com/brave/security/issues/new/choose) one https://github.com/brave/security/issues/762

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on
